src/Components/TransactionHandler/AbstractTransactionHandler.php

```diff
@@ -93,11 +96,18 @@ protected function executeRequest(array $request): array
         try {
             $response = $this->client->request($request);
 
-            $this->dataHandler->logResponse($this->paymentTransaction, $this->context, [
+            $this->transactionDataHandler->logResponse($this->paymentTransaction, $this->context, [
                 'request' => $request,
                 'response' => $response,
             ]);
 
+            $this->orderActionLogDataHandler->createOrderActionLog(
```

Comment:
is there a reason, why you have not implemented the `createOrderActionLog` method within the `transactionDataHandler`? 
Moving the method into the `transactionDataHandler` will prevent missing adding the logger in any further requests (if some added in the future)